src/components/EditableText.tsx

```diff
@@ -83,7 +84,7 @@ function EditableText({ saveValue, value, className, bold, truncate }: Props) {
   };
 
   return (
-    <Wrapper className={className}>
+    <Box variant="rowCenter" height="100%" {...props}>
```

Comment:
This breaks the alignment on `AccountList`
![image](https://user-images.githubusercontent.com/1561200/109637861-a07a4d00-7b4d-11eb-8634-da938e503da4.png)

Reply:
Fixed.